dev-db/sqlcl-bin/sqlcl-bin-19.4.0.354.0937.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN="${PN/-bin}"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="Oracle SQLcl is the new SQL*Plus"
+HOMEPAGE="https://www.oracle.com/technetwork/developer-tools/sql-developer/downloads/index.html"
+SRC_URI="${MY_P}.zip"
+RESTRICT="bindist fetch mirror"
+
+LICENSE="OTN"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND=""
+RDEPEND="virtual/jre:1.8
+	dev-java/java-config:2
+	dev-db/oracle-instantclient"
```

Comment:
Please sort in alphabetical order (ie just reverse the list). Sorry for these minor nitpicks after it's stood for so long, but let's make it look good! Can't really test it.

Reply:
No problems, I understand and share this point of view